src/scvmm/azext_scvmm/commands.py

```diff
@@ -142,17 +165,20 @@ def load_command_table(self: AzCommandsLoader, _):
         )
         g.custom_command('update', 'update_vm', supports_no_wait=True)
         g.custom_show_command('show', 'show_vm')
-        g.custom_command('list', 'list_vm')
+        g.custom_command('list', 'list_vm',
+                         deprecate_info=g.deprecate(redirect='az scvmm vm show', hide=True))
```

Comment:
```suggestion
                         deprecate_info=g.deprecate(redirect='scvmm vm show', hide=True))
```

Reply:
May I ask the reason for this redirection?

Reply:
The VMInstance is a child resource of Machine resource type. We don't have a direct API support in ARM for listing all the child resources in a  subscription or a resourceGroup. We could use ARG queries to do that, but that's not yet finalised.
If required, we'll revisit this command later, after discussing internally.

Reply:
Does the behavior of `list` command be changed, i.e., Was the listing feature not supported in the previous version?